pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -153,8 +153,8 @@ pub(super) async fn connection_manager_loop_step(
                             match new_state {
                                 // we're already active as walreceiver, no need to reactivate
                                 TimelineState::Active => continue,
-                                TimelineState::Broken | TimelineState::Stopping => {
-                                    debug!("timeline entered terminal state {new_state:?}, stopping wal connection manager loop");
+                                TimelineState::Broken { .. } | TimelineState::Stopping => {
+                                    info!("timeline entered terminal state {new_state:?}, stopping wal connection manager loop");
```

Comment:
```suggestion
                                    debug!("timeline entered terminal state {new_state:?}, stopping wal connection manager loop");
```

Reply:
Gotcha, I think I missed it during conflict resolution